server/base-routes.js

```diff
@@ -154,7 +154,7 @@ var baseRoutes = [
   }, {
     method: 'GET',
     path: '/api/polyfill.js',
-    handler: function(request, reply) {
+    handler: function(request, h) {
```

Comment:
What's h?

Reply:
Hapi's new request/response lifecycle uses this thing they call the "toolkit" and it's shortened to `h` 

A full porting guide is available here: https://github.com/hapijs/hapi/issues/3658 if you're curious

Reply:
ooook.